components/menu/menu-item-selectable-mixin.js

```diff
@@ -9,6 +9,11 @@ export const MenuItemSelectableMixin = superclass => class extends MenuItemMixin
 		};
 	}
 
+	constructor() {
+		super();
+		this.selected = false;
```

Comment:
@margaree @dbatiste I made this change to the actual component. Otherwise `this.selected` is `undefined` unless it's set.

Reply:
I've noticed a couple instances where the linter will complain about uninitialized booleans in certain statements.